### PR TITLE
Created a Gallery Page

### PR DESCRIPTION
**Pull request by andrewtedi**
This pull request will add a gallery page that will be accessed by all users for displaying pictures of club activities.

Features include:
- Image Carousel
- Gallery Scaling by Window Size
- Use of AWS Buckets for Retrieving and Displaying Gallery Images

All images will be added to the gallery using AWS S3 Buckets except for the ones displayed on the carousel, which are added locally in the "carouselImages" folder under the "images" folder. To test if these images will be displayed, S3 credentials will have to be set up in the config files on the local side. Then upload images to the S3 bucket set in the config files using the Upload Images feature on the SCE website.

Routing and testing for the gallery page and image retrieval are finished.

Possible changes that may need to be made:
- Gallery format
- Carousel Images

Gallery in 1920x1080 aspect ratio
gallery

### CODE REVIEW

**Review by aashrav — APPROVED** [review]
Awsome!!

File: src/Pages/Gallery/Gallery.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        <h2 className="header">Gallery</h2>
+        <Container className="container">
+          <CarouselSlider carouselPics={this.state.carouselPics} />
+          <div className="gallery">
+            {
+              this.state.picList.map((pic, index) => {
+                return (
+                  <img src={pic} className="singleImage"
+                    onClick={() => this.toggleModal(pic)}
+                    key={index} id={index} />
```
Comment (charlotte-zhuang) [inline src/Pages/Gallery/Gallery.js:69]: Potentially problematic use of index as key and id. Also, I believe React uses index as the key for you, so there's no need to specify `key={index}`.

**Review by charlotte-zhuang — COMMENTED** [review]
Looks good, except I didn't have time to actually test using an S3 connection. I think I can do a more thorough review later.

**Review by law-nguyen — APPROVED** [review]
It works fine for me

**Review by IsitaB — COMMENTED** [review]
wooooo great functionality! it works. The UI is a bit questionable. Change the font of `Gallery` maybe

**Review by IsitaB — COMMENTED** [review]
excellent quality of code. excellent functionality

could we make the contents wider? There's a lot of white space; it looks a little awkward. The title `Gallery` looks a bit awkward too

**Pull request closed without merging**